### PR TITLE
Explore search

### PR DESCRIPTION
**Pull request by laurieboyes**
screen shot 2016-02-23 at 17 30 58

### CODE REVIEW

File: client/components/explore/main.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+.topic-search__intro {
+	@include oTypographySans(s);
+	color: oColorsGetPaletteColor(cold-1);
+	text-align: center;
+}
+
+.topic-search__input {
+	@include oFormsCommonField();
+	@include oTypographySansData(m);
```
Comment (laurieboyes) [inline client/components/explore/main.scss]: We shouldn't _really_ be overriding origami component fonts, but Paul was very grateful

Comment (keirog) [inline client/components/explore/main.scss]: I'm not keen on this… Why override just this one and not other form styles?

Comment (keirog) [inline client/components/explore/main.scss]: Can we check with Alberto about changing it for all o-forms controls?

Comment (keirog) [inline client/components/explore/main.scss]: As per our offline chat, I'm backing down on this :see_no_evil:

**Comment by keirog** [comment]
:+1:

**Comment by phamann** [comment]
:v:

**Pull request merged**